client/src/entry/analysis/App.vue

```diff
@@ -108,6 +109,7 @@ export default {
                 return "Are you sure you want to leave the page?";
             }
         };
+        useEntryPointStore().ensurePollingEntryPoints();
```

Comment:
@dannon Where should this go? I think the app should always poll interactive entry points.

Reply:
Should work like the history store monitors contents -- it's not strapped onto app itself, but when you imort and start using it it sets up the polling.  (so, the action of using this store in whatever component would ensure this gets kicked off)